.github/workflows/montly-release.yaml

```diff
@@ -0,0 +1,90 @@
+on:
+  # Manual trigger mainly to create patch version
+  workflow_dispatch:
+    inputs:
+      patch_version:
+        description: 'to control the patch version of TinyTeX (i.e., z in x.y.z), so we could release patch versions like 2020.09.19 when necessary' 
+        required: false
+        type: string
+#  # Monthly release 
+#   schedule: 
+#     - cron: '10 2 1 * *'
```

Comment:
- [ ] To uncomment before merging

Reply:
same as above comment https://github.com/yihui/tinytex-releases/pull/22/files#r851056855

Reply:
I believe you meant the line below, but I get it. See my answer above.